### ISSUE TITLE
이미지 디헤이징 자료조사

### ISSUE THREAD

**Issue by d0gn** [d0gn/Driving-On-Weather#11]
이미지 디헤이징에 대한 자료조사

**Comment by meo-yeong** [comment]
https://ieeexplore.ieee.org/abstract/document/8967644
비와 안개에 대하여 이미지 디헤이징에 관한 논문

**Comment by mizzcereal** [comment]
https://hayunjong83.tistory.com/54
명암 보정 기술에 대한 논문리뷰 및 아래는 간단하게 정리한 내용

어두운 이미지의 광량 증가를 이미지의 특정적인 커브 예측 작업으로 표헌하는 zero-dce

DCE-Net(가벼운 심층 신경망)을 학습시켜 input image의 다이나믹레인지를 조정하는 픽셀 마다의 고차커브를 예측(이때 예측은 픽셀값의 범위,단조성,미분가능성을 고려)
학습과정에서 짝을 이루는 데이터 or 각각 대응되지 않는 집합데이터 필요없고 암묵적으로 광량 향상의 성능 측정 및 신경망의 학습을 이끌어내는 일련의 비참조 손실함수(non-reference Loss Function)을 통하여 학습

**Comment by jkjnaver** [comment]
https://kys0411.tistory.com/60?utm_source=chatgpt.com
Dark Channel Prior(이하 DCP)는 단일 이미지 안개제거 기법

**Comment by mizzcereal** [comment]
https://www.mdpi.com/1424-8220/23/3/1347

YOLO7보다 검출이 더 좋은 새로운 IDOD-YOLO7알고리즘 설명 논문
기존 안개제거 기법중 크게 나뉘는 4가지 방법인 1. 이미지 향상 기반 방법, 2. 물리 모델 기반 방법, 3. 심층 신경만 기반 방법, 4. 객체 검출과 결합된 방법대신 IDOD-YOLO7에서는 AOD모듈, SAIP모듈, YOLO7객체검출 네트워크를 활용해 이미지 디헤이징 진행

**Comment by jkjnaver** [comment]
https://scienceon.kisti.re.kr/commons/util/originalView.do?cn=JAKO201125736641056&oCn=JAKO201125736641056&dbt=JAKO&journal=NJOU00423616
TI DAVINCI를 이용한 영상 개선 알고리즘 구현 논문


TI DAVINCI DM6446에 적용해본 결과 720x480 영상 개선 적용 시 초당 15 화면의 성능을 보인다. 추가적인 최적호를 통해 속도를 더욱 개선할 수 있다. 향후 20 화면 이상의 속도를 내기 위해 카메라에 내장시키는 방안을 고려하고 있다.

**Comment by jkjnaver** [comment]
https://www.dbpia.co.kr/pdf/cpViewer
Image Dehazing Enhancement Algorithm Based on Mean Guided Filtering

이미지 복원 및 향상을 위한 알고리즘, 특히 평균 유도 필터링(Mean Guided Filtering)을 기반으로 한 이미지 디헤이징(Image Dehazing) 향상 기법에 대한 연구

**Comment by meo-yeong** [comment]
안개 제거 기본식에서의 t(x)에 관한 정보
https://www.slideshare.net/slideshow/precomputed-atmospheric-scattering-156956686/156956686

**Comment by meo-yeong** [comment]
sparse 사전학습
https://bskyvision.com/entry/%EC%8A%A4%ED%8C%8C%EC%8A%A4-%EC%82%AC%EC%A0%84-%ED%95%99%EC%8A%B5sparse-dictionary-learning%EC%9D%98-%EA%B9%8A%EC%9D%B4-%EC%9E%88%EC%9C%BC%EB%A9%B4%EC%84%9C%EB%8F%84-%EC%89%AC%EC%9A%B4-%EC%9D%B4%ED%95%B4#google_vignette
가우시안 혼합모델
https://untitledtblog.tistory.com/133
깨끗한 이미지를 위해서는 t(x)를 구하여야하는데 그방법에 관한 사이트이다.

**Comment by meo-yeong** [comment]
https://openaccess.thecvf.com/content_ICCV_2017/papers/Li_AOD-Net_All-In-One_Dehazing_ICCV_2017_paper.pdf
AOD NET

**Comment by meo-yeong** [comment]
Rain Streak Mask에관한자료들

https://github.com/Muccul/AddRain-CycleGAN

**Comment by meo-yeong** [comment]
https://www.kaggle.com/code/talevy23/cnn-with-parallel-convolution-layers
parallel convolution
병렬 합성곱 신경망

**Comment by meo-yeong** [comment]
https://github.com/walsvid/AOD-Net-PyTorch/blob/master/train.py
AOD-NET예제코드
model.py 확인바람 
우리가 찾은 deraining and dehazing 도 이런식으로 구현할 예정

**Comment by meo-yeong** [comment]
https://ieeexplore.ieee.org/document/8784948
image deraining
